integration_tests/mockApis/prisonerListApi.ts

```diff
@@ -0,0 +1,310 @@
+import { SuperAgentRequest } from 'superagent'
+import { stubFor } from './wiremock'
+
+const stubPrisonerList = (prisonId = 'BXI'): SuperAgentRequest =>
+  stubFor({
+    request: {
+      method: 'GET',
+      urlPattern: `/prisoner-search/prison/${prisonId}\\?page=0&size=9999`,
```

Comment:
There's a slightly better pattern you can use where there are query string parameters, which means you don't need them all on the path - its better because you can't always guarantee the order in which query string params are presented, you don't need to mess around with escaping the question mark, and you can apply a matcher (not just equality) on each query string parameter.
We have a couple of examples of this already in `curiousApi.ts`, but perhaps you could change the method sig and request matcher to this:
```
const stubPrisonerList = (prisonId = 'BXI', page = 0, pageSize = 999): SuperAgentRequest =>
  stubFor({
    request: {
      method: 'GET',
      urlPathPattern: `/prisoner-search/prison/${prisonId}`,
      quertParameters: {
        page: { equalTo: `${page}` },
        size: { equalTo: `${pageSize}` },
      }
},
```

Reply:
Much cleaner, I've made this change!